src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
```

Comment:
Rename local variable:

In general, it's not good to use single-character variable names, the only exception is for iterators.
Yes, here it's the case, and both **p** and **p1** are iterators, but since both also have too similar naming.
Renaming at least one of them would be much better for reading.
![image](https://github.com/pedro-ruas/mate-v2-js_average-ages/assets/54222238/bfab41dc-b0e6-4357-b725-3dd925871f29)